docs/Customizing_site.rst

```diff
@@ -4666,7 +4670,7 @@ to:
 .. |image4| image:: images/results_unifiedsearch_wikipedia.png
 .. |image5| image:: images/search_results_notwikipedia.png
 .. |image6| image:: images/page_layout/template.png
-.. |image7| image:: images/page_layout/template.png
+.. |image7| image:: images/page_layout/extension_structure.png
```

Comment:
Looks like you forgot to add the image

Reply:
No, the image is already there, but it was not referenced in the doc.